.storybook/preview-head.html

```diff
@@ -0,0 +1,4 @@
+<!-- Copyright © 2014 Monotype Imaging Inc. All rights reserved -->
+<link href="https://cdn.jsdelivr.net" rel="preconnect" crossorigin="anonymous">
+<link href="https://cdn.jsdelivr.net/npm/boosted@5.0.0/dist/css/boosted.min.css" rel="stylesheet" integrity="sha384-9BDn6EpdWMmf91d+gsoVk3n0CHQOmr5P8sdm6cZWKElGfFzZjkVrILeQfCaQde9L" crossorigin="anonymous">
```

Comment:
I think you should be able to use sole shortcut to the latest v5, from JSdelivr : something like `boosted@5`. This way you'd stay up-to-date with npm for free.

Reply:
😲 Seems logic but I hadn't thought about it. Thx @ffoodd!

Reply:
Hum, but it also means that we need to remove the `integrity` like that:
```html
<link href="https://cdn.jsdelivr.net/npm/boosted@5/dist/css/boosted.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/boosted@5/dist/js/boosted.bundle.min.js" crossorigin="anonymous"></script>
```
Given the fact that it is only used in the context of Storybook (and Zeroheight via an `<iframe>`) and not in a real application/website, I would say this security matter it is not that important. Not 100% sure about that. What do you think?

Reply:
Agreed, as long as Boosted npm package depends on this repo I think it's sage 👌